packages/scrollable_positioned_list/test/scrollable_positioned_list_test.dart

```diff
@@ -37,17 +37,21 @@ void main() {
     tester.binding.window.devicePixelRatioTestValue = 1.0;
     tester.binding.window.physicalSizeTestValue =
         const Size(screenWidth, screenHeight);
+    itemCount = itemCount ?? defaultItemCount;
```

Comment:
can you just change line 29 to 

int itemCount = defaultItemCount

and remove this line?